src/codemagic/tools/app_store_connect/action_groups/bundle_ids_action_group.py

```diff
@@ -0,0 +1,165 @@
+
+    @cli.action(
+        "get",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("get-bundle-id", "0.49.0"),
+    )
+    def get_bundle_id(
+        self,
+        bundle_id_resource_id: ResourceId,
+        should_print: bool = True,
+    ) -> BundleId:
+        """
+        Get specified Bundle ID from Apple Developer portal
+        """
+
+        return self._get_resource(bundle_id_resource_id, self.api_client.bundle_ids, should_print)
+
+    @cli.action(
+        "list",
+        BundleIdArgument.BUNDLE_ID_IDENTIFIER_OPTIONAL,
+        BundleIdArgument.BUNDLE_ID_NAME,
+        BundleIdArgument.PLATFORM_OPTIONAL,
+        BundleIdArgument.IDENTIFIER_STRICT_MATCH,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("list-bundle-ids", "0.49.0"),
+    )
+    def list_bundle_ids(
+        self,
+        bundle_id_identifier: Optional[str] = None,
+        bundle_id_name: Optional[str] = None,
+        platform: Optional[BundleIdPlatform] = None,
+        bundle_id_identifier_strict_match: bool = False,
+        should_print: bool = True,
+    ) -> List[BundleId]:
+        """
+        List Bundle IDs from Apple Developer portal matching given constraints
+        """
+
+        def predicate(bundle_id):
+            return bundle_id.attributes.identifier == bundle_id_identifier
+
+        bundle_id_filter = self.api_client.bundle_ids.Filter(
+            identifier=bundle_id_identifier,
+            name=bundle_id_name,
+            platform=platform,
+        )
+        bundle_ids = self._list_resources(
+            bundle_id_filter,
+            self.api_client.bundle_ids,
+            should_print,
+            filter_predicate=predicate if bundle_id_identifier_strict_match else None,
+        )
+
+        return bundle_ids
+
+    @cli.action(
+        "delete",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_ID,
+        CommonArgument.IGNORE_NOT_FOUND,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("delete-bundle-id", "0.49.0"),
+    )
+    def delete_bundle_id(
+        self,
+        bundle_id_resource_id: ResourceId,
+        ignore_not_found: bool = False,
+    ) -> None:
+        """
+        Delete specified Bundle ID from Apple Developer portal
+        """
+
+        self._delete_resource(self.api_client.bundle_ids, bundle_id_resource_id, ignore_not_found)
+
+    @cli.action(
+        "profiles",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_IDS,
+        ProfileArgument.PROFILE_TYPE_OPTIONAL,
+        ProfileArgument.PROFILE_STATE_OPTIONAL,
+        ProfileArgument.PROFILE_NAME,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("list-bundle-id-profiles", "0.49.0"),
+    )
+    def list_bundle_id_profiles(
```

Comment:
Previously defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L782-L823